packages/flutter/test/widgets/async_test.dart

```diff
@@ -211,6 +251,24 @@ void main() {
       ]));
       expect(find.text('AsyncSnapshot(ConnectionState.none, null, null)'), findsNWidgets(2));
     });
+    testWidgets('when initialData is used with null Future and Stream', (WidgetTester tester) async {
+      await tester.pumpWidget(new Column(children: <Widget>[
+        new FutureBuilder<String>(future: null, builder: snapshotText, initialData: 'I',),
```

Comment:
[nit] Avoid trailing comma for single line parameter lists.

Reply:
Trailing commas removed.